src/pieces/commands/autocommit/autocommit.py

```diff
@@ -126,40 +122,47 @@ def git_commit(**kwargs):
         # Remove leading and trailing whitespace
         commit_message = commit_message.strip()
 
-
-        # Issues
-        repo_details = get_git_repo_name()
-        issues = get_repo_issues(*repo_details) if repo_details else [] # Check if we got a vaild repo name
-
-        if issues:
-            # Make the issues look nicer
-            issue_list = [
-                f"- `Issue_number: {issue['number']}`\n- `Title: {issue['title']}`\n- `Body: {issue['body']}`"
-                for issue in issues
-            ]
-            issue_list = "\n".join(issue_list) # To string
+        if issue_flag:
             
-            try:
-                
-                issue_number = pos_client.QGPTApi(api_client).relevance(
-                        pos_client.QGPTRelevanceInput(
-                            query=issue_prompt.format(issues=issue_list),
-                            paths=paths,
-                            application=commands_functions.application.id,
-                            model=model,
-                            options=pos_client.QGPTRelevanceInputOptions(question=True)
-                        )).answer.answers.iterable[0].text
-        
+            issue_prompt = """Please provide the issue number that is related to the changes, If nothing related write 'None'.
```

Comment:
Just to verify, you can also manually insert the issue number right? The default we're using is 3.5 turbo I believe. To get consistent results from this at production level we'd have to use function calling from gpt-3.5-turbo-0613. Claude now offers function calling as well.

Reply:
@BishoyHanyRaafat talked with the team, currently Pieces doesn't have the ability for function calling. Would you be opposed to adding an option for the user to manually override whatever chatgpt comes back with and write their own message?

Reply:
What we should do to resolve that issue??